src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/utils/validators/UpdateSessionTemplateValidator.kt

```diff
@@ -17,23 +15,20 @@ import java.time.LocalDate
 @Component
 class UpdateSessionTemplateValidator(
   private val visitRepository: VisitRepository,
-  private val sessionTemplateRepository: SessionTemplateRepository,
   private val sessionLocationMatcher: SessionLocationMatcher,
   private val sessionCategoryMatcher: SessionCategoryMatcher,
   private val sessionIncentiveLevelMatcher: SessionIncentiveLevelMatcher,
   private val sessionTemplateUtil: SessionTemplateUtil,
   private val sessionTemplateMapper: SessionTemplateMapper,
-  @Value("\${policy.session.booking-notice-period.maximum-days:28}")
-  private val policyNoticeDaysMax: Long,
 ) {
-
   fun validate(sessionTemplate: SessionTemplateDto, updateSessionTemplateDto: UpdateSessionTemplateDto): List<String> {
     val errorMessages = mutableListOf<String>()
     val hasVisits = visitRepository.hasVisitsForSessionTemplate(sessionTemplate.reference)
     validateUpdateSessionTemplateTime(sessionTemplate, updateSessionTemplateDto, hasVisits)?.let { errorMessages.add(it) }
     validateUpdateSessionTemplateDate(sessionTemplate, updateSessionTemplateDto, hasVisits).let { errorMessages.addAll(it) }
     validateUpdateSessionTemplateWeeklyFrequency(sessionTemplate, updateSessionTemplateDto, hasVisits)?.let { errorMessages.add(it) }
-    validateUpdateSessionCapacity(sessionTemplate, updateSessionTemplateDto, hasVisits).let { errorMessages.addAll(it) }
+
+    // removed session capacity validation
```

Comment:
If leaving a comment, can you put the ticket number or reason it was removed. As we won't remember in future haha

Reply:
removed comment